network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/airtable_block.html

```diff
@@ -1 +1,10 @@
-<script async defer src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe class="airtable-embed mb-4 airtable-dynamic-height" src="{{ value.url }}" frameborder="0" onmousewheel="" width="100%" height="{{ value.height }}" style="background: transparent; border: 1px solid #ccc;"></iframe>
+<div class="container {{block.block_type}}-block">
+    <div class="row justify-content-center no-gutters">
+        <div class="streamfield-content">
+            <script async defer src="https://static.airtable.com/js/embed/embed_snippet_v1.js"></script><iframe
+                class="airtable-embed mb-4 airtable-dynamic-height" src="{{ value.url }}" frameborder="0"
+                onmousewheel="" width="100%" height="{{ value.height }}"
+                style="background: transparent; border: 1px solid #ccc;"></iframe>
+        </div>
+    </div>
+</div>
```

Comment:
We need to check if this block is even used anywhere. I don't recall seeing it, but maybe I've just skipped over it.

Reply:
Hi Kalob, I also have not seen it being used anywhere. I think Ben has also checked and confirmed that he could not find any use of it in the block-finding tool. Does this mean we should remove the block? completely from the application? If so, I can make a ticket 👍

Reply:
I think this is a ticket now, or at least it's in discussion. I'll resolve this thread. Thanks Daniel!